### PR TITLE
Ds 573 tokens improvements and fixes

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/helpers/tokens/inlineMessage.json [hunk 1]
```diff
@@ -11,6 +11,26 @@
...
+          "value": "{orange.500}",
+          "type": "color"
+        },
+        "iconInfo": {
+          "value": "{blue.500}",
+          "type": "color"
+        },
+        "iconError": {
+          "value": "{error.500}",
+          "type": "color"
```
Comment (monikaracka) [inline packages/design-system/src/helpers/tokens/inlineMessage.json:33]: I think icons should be in different places in this JSON file e.g., for success icon
```
"success: {
   "color":{
      "background":{...},
      "text": {
         "default":{...},
         "icon":{
            "value": "{primary.500}",
            "type": "color"
}}}
```
So token will be `text-inlineMessage-success-icon`

File: packages/design-system/src/components/InlineMessage/index.tsx [hunk 2]
```diff
@@ -19,17 +19,17 @@ export const InlineMessage: FC<InlineMessageProps> = ({
...
-    default: "text-neutral-900",
-    success: "text-primary-500",
-    warning: "text-orange-500",
-    info: "text-blue-500",
-    error: "text-error-500",
+    default: "text-inlineMessage-base-iconDefault",
+    success: "text-inlineMessage-base-iconPrimary",
+    warning: "text-inlineMessage-base-iconWarning",
+    info: "text-inlineMessage-base-iconInfo",
+    error: "text-inlineMessage-base-iconError",
```
Comment (monikaracka) [inline packages/design-system/src/components/InlineMessage/index.tsx:26]: I added a comment to the JSON file about these tokens.

File: packages/design-system/src/components/InputText/index.tsx [hunk 3]
```diff
@@ -34,7 +40,7 @@ export const InputText = ({ text, type, labelHtmlFor, className }: InputTextProp
   return (
     <div
       className={classNames(
-        "text-inputText-fontBase m-0 min-h-[6px] p-0 empty:hidden",
+        "text-inputText-fontBottomText p-inputText-paddingBase m-0 min-h-[6px] empty:hidden",
```
Comment (monikaracka) [inline packages/design-system/src/components/InputText/index.tsx:43]: Why did you change font token name? Now here is `paddingBase` and `fontBottomText`

File: packages/design-system/src/components/Stepper/index.tsx [hunk 4]
```diff
@@ -86,7 +91,6 @@ const StepperComponents = {
       >
         <p
           className={classNames(
-            "text-stepper-fontTitle font-stepper-weightTitle empty:hidden",
```
Comment (monikaracka) [inline packages/design-system/src/components/Stepper/index.tsx:89]: Yoou shouldn't remove `font-stepper-weightTitle empty:hidden`

File: packages/design-system/src/helpers/tokens/radio.json [hunk 5]
```diff
@@ -43,6 +43,10 @@
         "error": {
           "value": "{error.500}",
           "type": "color"
+        },
+        "disabled": {
+          "value": "{error.500}",
```
Comment (monikaracka) [inline packages/design-system/src/helpers/tokens/radio.json:48]: I think it's not `error.500` 😉

File: packages/design-system/src/helpers/tokens/rating.json [hunk 6]
```diff
@@ -99,6 +99,16 @@
       }
     }
   },
+  "gapAlignLedt": {
```
Comment (monikaracka) [inline packages/design-system/src/helpers/tokens/rating.json:102]:
```suggestion
  "gapAlignLeft": {
```

File: packages/design-system/src/components/InputText/index.tsx (hunk 3 above)
Comment (zuzannanap) [reply packages/design-system/src/components/InputText/index.tsx:43]: because now you can independently change the size of the label and supporting text and I think it might be useful

Comment (monikaracka) [reply packages/design-system/src/components/InputText/index.tsx:43]: I meant the difference between padding**Base** and font**BottomText**. You changed `fontBase` to `fontBottomText`.

**Review by monikaracka — APPROVED** [review]
